test/model/mapping/coercer_test.rb

```diff
@@ -16,22 +17,41 @@
       coercer.to_record(entity).must_equal(name: 'Tyrion Lannister')
     end
 
+    describe 'new record' do
+      # Bug:
+      # https://github.com/lotus/model/issues/155
+      it 'ignores unset values' do
+        entity = User.new(name: 'Daenerys Targaryen')
+        coercer.to_record(entity).must_equal(name: 'Daenerys Targaryen')
+      end
+
+      it 'forces nil values' do
+        entity = User.new(name: 'Daenerys Targaryen', age: nil)
+        coercer.to_record(entity).must_equal(name: 'Daenerys Targaryen')
```

Comment:
I think we should not ignore the attributes when explicitly set to nil, even when creating a new record. The test case above should something like:
```ruby
entity = User.new(name: 'Daenerys Targaryen', age: nil)
coercer.to_record(entity).must_equal(name: 'Daenerys Targaryen', age: nil)
```